src/main/java/org/pac4j/demo/dw/Pac4JDemoApplication.java

```diff
@@ -45,6 +47,12 @@ protected void configure() {
         });
 
         env.jersey().register(ViewsResource.class);
-    }
 
+        env.healthChecks().register(getName(), new HealthCheck() {
+            @Override
+            protected Result check() throws Exception {
+                return Result.healthy();
```

Comment:
by the way, what could be nice here is to have some healthchecks to verify that the token used for the SAML and the facebook client are valid. Not sure how, and even maybe we could start thinking about automatically doing that in dropwizard-pac4j!